Scripts/script-ReadFile.yml

```diff
@@ -0,0 +1,33 @@
+commonfields:
+  id: eccea40d-921c-4a52-875a-738855531f58
```

Comment:
static ID
Also please limit by size, if someone will load a few Giga file here we would die